releases/release-1.26/release-notes/maps/pr-113719-map.yaml

```diff
@@ -0,0 +1,3 @@
+pr: 113719
+releasenote:
+  text: bump runc to v1.1.4.
```

Comment:
does this need to be `runc` and `v1.1.4`?

Reply:
Yes. I agree. Thanks.